noir-projects/aztec-nr/aztec/src/messages/logs/note.nr

```diff
@@ -44,20 +34,24 @@ pub fn compute_partial_note_log<Note>(
 where
     Note: NoteType + Packable,
 {
-    compute_log(
+    let ciphertext = compute_ciphertext(
         note,
         storage_slot,
         recipient,
         PARTIAL_NOTE_PRIVATE_MSG_TYPE_ID,
-    )
+    );
+
+    let log = prefix_with_tag(ciphertext, recipient);
+
+    log
 }
 
-fn compute_log<Note>(
+fn compute_ciphertext<Note>(
     note: Note,
     storage_slot: Field,
     recipient: AztecAddress,
     msg_type: u64,
-) -> [Field; PRIVATE_LOG_SIZE_IN_FIELDS]
+) -> [Field; PRIVATE_LOG_CIPHERTEXT_LEN]
```

Comment:
Feels like we will need to fix the logs vs message naming because in the past we had no offchain messages and now we have incorrect naming because this ciphertext goes to message as well. Hence the `PRIVATE_LOG_CIPHERTEXT_LEN` constant name is a kinda stale.

Actually not super sure if it's stale because the private logs are what places the size constraint here and we use the same size for messages as well just because it's convenient. Probably best to not deal with now and see what the future brings.

Reply:
Since these renamings are quick to implement decided to just had a stab at it in [this PR](https://github.com/AztecProtocol/aztec-packages/pull/16882).